diff --git a/resources/views/layouts/app.blade.php b/resources/views/layouts/app.blade.php
index d72fe47..2a54e2f 100644
--- a/resources/views/layouts/app.blade.php
+++ b/resources/views/layouts/app.blade.php
@@ -1,20 +1,13 @@
 <!DOCTYPE html>
 <html lang="{{ config('app.locale') }}">
 <head>
+	<!-- Messrs David Mesman, Robin Kuijpers, Martijn van Halderen & Ted van Riel Proudly present Zoetermeer-Vooruit's new website. -->
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
-    <!-- CSRF Token -->
     <meta name="csrf-token" content="{{ csrf_token() }}">
-
     <title>{{ config('app.name', 'Laravel') }}</title>
-
-    <!-- Styles -->
     <link href="{{ asset('css/app.css') }}" rel="stylesheet">
-
-
-    <!-- Scripts -->
     <script>
         window.Laravel = {!! json_encode([
             'csrfToken' => csrf_token(),
diff --git a/resources/views/welcome.blade.php b/resources/views/welcome.blade.php
index 3f3990a..a7e9add 100644
--- a/resources/views/welcome.blade.php
+++ b/resources/views/welcome.blade.php
@@ -18,7 +18,6 @@
     });
 
     </script>
-
 <header>
   <center>
   <div class="dark">
diff --git a/routes/web.php b/routes/web.php
index ec9973a..5728073 100644
--- a/routes/web.php
+++ b/routes/web.php
@@ -10,33 +10,43 @@
 |
 */
 
-Route::get("contact", "RoutingController@contact");
-Route::get("welcome", "RoutingController@landing");
-Route::get("/", "RoutingController@landing");
-Route::get('/home/edit', 'RoutingController@editHome')->middleware('admin');
-Route::post('/home/edit', 'RoutingController@saveHTML')->middleware('admin');
-Route::get('/about/edit', 'RoutingController@editAbout')->middleware('admin');
-Route::post('/about/edit', 'RoutingController@saveHTMLAbout')->middleware('admin');
-Route::get("about", "RoutingController@about");
+Route::get("contact", 		"RoutingController@contact"			);
+Route::get("welcome", 		"RoutingController@landing"			);
+Route::get("/", 			"RoutingController@landing"			);
+Route::get('/home/edit', 	'RoutingController@editHome'		)->middleware('admin');
+Route::get('/about/edit', 	'RoutingController@editAbout'		)->middleware('admin');
+Route::get("about", 		"RoutingController@about"			);
+Route::post('/about/edit', 	'RoutingController@saveHTMLAbout'	)->middleware('admin');
+Route::post('/home/edit', 	'RoutingController@saveHTML'		)->middleware('admin');
 
 Route::group(['prefix'=>'quiz'], function() {
-    Route::get('/', 'QuizController@view');
-    Route::get('/{id}', 'QuestionController@view')->where('id', '[0-9]+');
-    Route::post('/delete', 'QuizController@delete');
-    Route::get('create', 'QuizController@add');
-    Route::post('create', 'QuizController@store');
-    Route::get('{id}/make', 'QuizController@make')->where('id', '[0-9]+');
-    Route::post('{id}/make', 'ReplyController@make')->where('id', '[0-9]+');
-    Route::get("{id}/statistics", "RoutingController@statistics")->where('id', '[0-9]+');
-    Route::post('{id}/statistics', 'QuizController@statistics')->where('id', '[0-9]+');
+    Route::get('/', 				'QuizController@view'			);
+    Route::get('/{id}', 			'QuestionController@view'		)->where('id', '[0-9]+');
+    Route::get('create', 			'QuizController@add'			);
+    Route::get('{id}/make', 		'QuizController@make'			)->where('id', '[0-9]+');
+    Route::get('{id}/statistics', 	'RoutingController@statistics'	)->where('id', '[0-9]+');
+    Route::post('create', 			'QuizController@store'			);
+    Route::post('{id}/make', 		'ReplyController@make'			)->where('id', '[0-9]+');
+    Route::post('/delete', 			'QuizController@delete'			);
+    Route::post('{id}/statistics', 	'QuizController@statistics'		)->where('id', '[0-9]+');
 });
 
 Route::group(['prefix'=>'question'], function(){
-	Route::get('/', 'QuestionController@view');
-	Route::get('{id}', 'QuestionController@form')->where('id', '[0-9]+');
-    Route::post('/{id}', 'QuestionController@edit')->where('id', '[0-9]+');
+	Route::get('/', 	'QuestionController@view');
+	Route::get('{id}', 	'QuestionController@form')->where('id', '[0-9]+');
+    Route::post('/{id}','QuestionController@edit')->where('id', '[0-9]+');
+});
+
+Route::group(['prefix' => 'users'], function() {
+	Route::get('/', 			'UserController@list'	)->middleware('admin');
+	Route::get('/{id}/admin', 	'UserController@promote')->middleware('admin');
+	Route::get('/{id}/demote', 	'UserController@demote'	)->middleware('admin');
+	Route::get('/{id}/allow', 	'UserController@allow'	)->middleware('admin');
+	Route::get('/{id}/ban', 	'UserController@ban'	)->middleware('admin');
 });
 
 Route::get('/home', 'HomeController@index');
 
 Auth::routes();
+
+Route::get('/{page}', 'CMSController@index');
\ No newline at end of file
